docs/CHANGELOG.asciidoc

```diff
@@ -10,6534 +10,421 @@
-Operations::
-* Fixes the logging destination for systemd installations and and updates the sysv filename to be consistent with other stack products {kibana-pull}74896[#74896]
-* Create keystore after installation {kibana-pull}76465[#76465]
-* Wrap blocked paths in quotes {kibana-pull}83560[#83560]
-Platform::
-* Adds loading indicator during debounce time {kibana-pull}80158[#80158]
-* Mark task as failed if maxAttempts has been met. {kibana-pull}80681[#80681]
-* Reactively disable Task Manager lifecycle when core services become unavailable {kibana-pull}81779[#81779]
-* Schedule retry based on schedule on recurring tasks {kibana-pull}83682[#83682]
-Querying & Filtering::
-* KQL autocomplete suggestions for field values will now have backslashes properly escaped {kibana-pull}85457[#85457]
-Security::
-* Prevent Kerberos and PKI providers from initiating a new session for unauthenticated XHR/API requests {kibana-pull}82817[#82817]
-* Fixes incomplete client cert chain when using PKI authentication with the login selector {kibana-pull}88229[#88229]
-Uptime::
-* Monitor status alert use url as instance {kibana-pull}81736[#81736]
-* Removes Connector flyouts after usage {kibana-pull}82126[#82126]
-* User experience display low values {kibana-pull}86026[#86026]
-* Simple monitor status alert fix for page duty and other connectors {kibana-pull}87460[#87460]
-* Fixes kuery bar dark theme {kibana-pull}87827[#87827]
-* Clear ping state when PingList component in unmounted {kibana-pull}88321[#88321]
-* UX use replace history instead of push on first load {kibana-pull}88586[#88586]
-* Fixes impacted page load errors {kibana-pull}88597[#88597]
+[[breaking-changes-7.12.0]]
+=== Breaking changes
 
-[[release-notes-7.10.2]]
-== {kib} 7.10.2
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 7.12.0, review the breaking changes, then mitigate the impact to your application.
 
-For detailed information about the 7.10.2 release, review the following bug fixes. For the breaking changes, refer to the <<breaking-changes-7.10,breaking changes in 7.10>>.
+// tag::notable-breaking-changes[]
 
-[float]
-[[security-update-v7.10.2]]
-=== Security update
-*Vega* visualizations are susceptible to stored and reflected XSS via a vulnerable version of the Vega library. When you create *Vega* visualizations or create a vulnerable URL that describes the visualization, an arbitrary JavaScript can execute in your browser.
+[discrete]
+[[breaking-89632]]
+.Removes geo threshold alert type
+[%collapsible]
+====
+*Details* +
+Geo containment alerts and the Geo threshold alert type have similar functionality, so the Geo threshold alert type has been removed. For more information, refer to {kibana-pull}89632[#89632].
 
-[float]
-[[affected-versions-v7.10.2]]
-==== Affected versions
-Affected versions include 7.10.1 and earlier.
+*Impact* +
+Use Geo containment alerts.
+====
+      
```

Comment:
```suggestion

[discrete]
[[breaking-so-upgrade-migrations]]
.New saved object upgrade migrations algorithm
[%collapsible]
====
*Details* +
When upgrading to 7.12.0, {kib} will use a new algorithm for performing saved object migrations. The new algorithm  enables {kib} to retry a failed migration without any user intervention. This can reduce downtime caused by temporary errors such as a bad network or temporarily overloaded {es} cluster because the upgrade migration will be able to proceed as soon as the underlying problem is resolved. For more information see <<upgrade-migrations, saved object upgrade migrations>>
*Impact* +
To rollback to an earlier version after upgrading to 7.12.0 requires different steps to previous versions of {kib}. Before upgrading, please carefully review the documentation for <<upgrade-migrations-rolling-back, rolling back to a previous version of {kib}>>
====

```